nixos/tests/all-tests.nix

```diff
@@ -273,6 +273,7 @@ in {
   krb5 = discoverTests (import ./krb5 {});
   ksm = handleTest ./ksm.nix {};
   kubernetes = handleTestOn ["x86_64-linux"] ./kubernetes {};
+  languagetool = handleTest ./languagetool.nix {};
```

Comment:
this test could also be added to `passthru.tests` on the languageserver derivation :)

Reply:
I'll add it.